src/types/options.ts

```diff
@@ -0,0 +1,168 @@
+export interface CoinpaymentsGetBasicInfoOpts {}
+
+export interface CoinpaymentsRatesOpts {
+  short?: number
+  accepted?: number
+}
+
+export interface CoinpaymentsCreateTransactionOpts {
+  currency1: string
```

Comment:
it is not currencyFrom and currencyTo?

Reply:
You can check the official docs :) not fixing someones API design/field.